lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -1317,6 +1337,39 @@ def __get_vnic_profile_id(self, nic):
                 )
             )
 
+    def __attach_numa_node(self, entity):
```

Comment:
This method doesn't handle the update/remove, so it's not fully idempotent, but I think we can go with that for start and improve later. If it's OK for High Perf VM.

Reply:
Right, I thought about that. So need to keep it in mind and  support that in the future.